### ISSUE TITLE
[Feature]: Bundle for economics/econometrics content

### ISSUE THREAD

**Issue by valerievossen** [tilburgsciencehub/website#859]
### Is your feature request related to a problem? Please describe.

I think the content for an economics student (who has less affinity with data than programs like marketing analytics, data science etc) is hard to find on the website.

### Describe the solution you'd like

It could be an idea to create something that gives you all the content in a bundle. Maybe a tutorial on doing good economic analysis, or a tutorial on how to do empirical research for your thesis for example. This would then point you towards all the (still upcoming) causal inference content. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by lachlandeer** [comment]
any updates on this @valerievossen ?

**Comment by valerievossen** [comment]
Not really, in the updated website there is a clear section on causal inference already, and we also moved away from the tutorial format. I will keep it in mind the coming weeks and think about what kind of content on research in an Economics thesis can be added. If you have any suggestions on this, I would be happy to hear them!
